terraform/environments/prd/aws/main.tf

```diff
@@ -17,6 +17,15 @@ provider "aws" {
   region = "eu-west-2"
 }
 
+module "sg" {
+  source = "../../../modules/aws/sg/"
+  vpc_id = module.vpcs["gigachad-tf"].vpc_id
```

Comment:
Everything looks cool man. One small issue is that i have this 
```
Releasing state lock. This may take a few moments...
╷
│ Error: Unsupported attribute
│ 
│   on main.tf line 22, in module "sg":
│   22:   vpc_id = module.vpcs["gigachad-tf"].vpc_id
│     ├────────────────
│     │ module.vpcs["gigachad-tf"] is a object
│ 
│ This object does not have an attribute named "vpc_id".
```

Reply:
Well, it is strange, but i found some workarond:
For some reasons, vpc_id does not output within object "gigachad-tf"
However, when I deployed only vpc module once, and destroyed it after, now vpc_id output works well, and I can deploy my new module "from scratch"

Reply:
Yeah, forget to add vpc_id output in output.tf
Fixed, now works well!